ideas/311-wallet.md

```diff
@@ -0,0 +1,204 @@
+
+- Usability issues that prevent simple transaction and erode trust
+  - Usability issues of our wallet specifically (e.g. how much am I sending, what is the value in fiat when sending via wallet and via chat)
+  - Usability issues of Ethereum itself (e.g. what does it mean 5 confirmations, was it sent or now? what is gas?)
+ 
+- Lack of killer features that drive adoption
+  - Lack of a goal-driven interaction flow. E.g. buy SNT/Cryptokitty
+  - For non crypto users; a lack of introduction or guidance. “I'll look into that later because I don't know what I'm doing.” E.g. what would a gentle transition from a DApp to wallet look like.
+
+### Why
+
+> Will solving this problem advance the goals of Status and adhere to its principles?
+
+Status aims to be a gateway to ethereum, and wallet and transactions are at the core of this ambition. Many aspects of the Status vision including the whitepaper depend on a functioning wallet that is secure, private, and easy to use, therefore it is important that Wallet creates a core foundation of usability and delight through best-in-class UI and compelling features.
+
+Furthermore, the central Wallet feature - transaction signing - is essential for dapp usage within Status , and vice versa.
+
+Solving usability bottlenecks and introducing killer features that give users a reason to open Status will ensure that Status drives adoption and will unblock more advanced use cases outlined in the whitepaper and beyond.
+
+### Success Criteria
+
+> How will we know when this problem has been addressed? What are qual and quant metrics that will help evaluate?
+
+Success for wallet is having the best mobile user experience for transactions, is super easy to use, offers compelling features, and is the best way to easily get tokens.  A fun onboarding experience is essential to give new users something to do and people to connect with.
+
+- 80%+ of UXR subjects report a positive experience when transacting
+- 5+ organic reports of users using Status for transactions (eg tweets, sightings, etc)
+- 3+ way for users to acquire tokens
+
+
+## Communication
+
+- `status channel`: [#status-core-wallet](https://get.status.im/chat/public/status-core-wallet)
+- `sync frequency`: Weekly Sync
+- Github:
+  - Project boards
+    - [Wallet epic project board](https://github.com/orgs/status-im/projects/48)
+    - [Maintenance project board](https://github.com/orgs/status-im/projects/24)
+  - Label: [wallet](https://github.com/status-im/status-react/labels/wallet)
+
+## Research
```

Comment:
What does this mean in terms of absolute days? Is the idea to treat these as mini projects each with their own research>spec>impl phase? Or are these more like individual issues that all come from one research phase that identifies P0 and P1 problems?